src/Cthulhu.jl

```diff
@@ -212,7 +212,7 @@ end
 function _descend(term::AbstractTerminal, interp::CthulhuInterpreter, mi::MethodInstance;
     override::Union{Nothing,InferenceResult}=nothing, debuginfo::Union{Symbol,DebugInfo}=DInfo.compact, # default is compact debuginfo
     params=current_params(), optimize::Bool=true, interruptexc::Bool=true,
-    iswarn::Bool=false, verbose=true, inline_cost::Bool=false)
+    iswarn::Bool=false, hide_type_stable::Bool=false, inline_cost::Bool=false)
```

Comment:
If we want to make this non-erroring for anyone who supplies `verbose` directly, then perhaps use
```julia
verbose=nothing, hide_type_stable=nothing, ...
```
and then later
```julia
if !(verbose === nothing || hide_type_stable === nothing)
    error("`verbose` is being replaced with `hide_type_stable`, do not use both")
end
if hide_type_stable === nothing
    hide_type_stable = something(verbose, false)
end
```
although even better would be to `depwarn` the use of `verbose`.

I'm not insisting on this; UI changes are kind of a gray zone as far as I am concerned. If others think this is more hassle than desirable, I'm fine with the way you've done it.

Reply:
Yeah, that should be better.

Reply:
We also want to keep supporting <kbd>v</kbd> ... ? I'd like to avoid it, since I guess it will complicate the UI.

Reply:
Yes, dropping support for that should be fine.